Packs/GenericSQL/pack_metadata.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
@amshamah419 can I ask you to explain orel:
1. the usage of this file in the upcoming Demisto version
2. how she should fill it

Reply:
Hi @ohaim1008! This file is used in the next version of Demisto to display the information of the pack in the marketplace. I'll include a template you can use:

```json
{
  "name": "Display name of the pack",
  "description": "Short description of what the pack contains/does",
  "support": "Cortex XSOAR",
  "serverMinVersion": "5.0.0",
  "currentVersion": "1.0.0",
  "author": "Cortex XSOAR",
  "url": "https://www.paloaltonetworks.com/cortex",
  "email": "",
  "categories": ["The category defined in the yml. e.g Network Security"],
  "tags": [],
  "created": "2020-03-10T08:37:18Z",
  "updated": "2020-03-10T08:37:18Z",
  "beta": false,
  "deprecated": false,
  "certification": "certified",
  "useCases": [],
  "keywords": [],
  "price": 0,
  "dependencies": {}
} 
```

If you have any questions, let me know.